src/Serilog/Core/Sinks/SafeAggregateSink.cs

```diff
@@ -18,6 +18,8 @@ class SafeAggregateSink : ILogEventSink
 {
     readonly ILogEventSink[] _sinks;
 
+    internal static SafeAggregateSink Empty = new(new ILogEventSink[0]);
```

Comment:
This appears to be unused?

Reply:
`sink ??= SafeAggregateSink.Empty;` in another file.